scripts/gen_docs.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -e
+
+# Author: Thomas Bechtold <tbechtold@suse.com>
+# License: Apache-2.0
+
+# List of project for running the doc generation
+: ${PROJECTS:="nova horizon cinder glance neutron manila aodh barbican ceilometer designate heat ironic magnum murano neutron-fwaas neutron-lbaas sahara"}
+# swift docs don't build successfully on stable/rocky
+# keystone docs dont build successfully with the script, requires python3, which should be whats being used
+# directory where the OpenStack git repositories are in
+: ${OPENSTACK_REPO_DIR:=${PWD}/devel/openstack}
```

Comment:
I don't see a reason to create two levels of directory structure above the additional ones that will be created by cloning the various projects. But if you decide to keep these, then the default output dir should also be within this directory structure as well (e.g. ${PWD}/devel/openstack-docs-html) to avoid creating **two** new deeply nested directory structures.

Reply:
put them both under devel for now

Reply:
Thanks

Reply:
`devel` is a bit too vague for my tastes.  How about `git` or `upstream` or `upstream-git`?

Reply:
changed to upstream